src/app/questions/file/file.component.ts

```diff
@@ -113,4 +125,30 @@ export class FileComponent implements ControlValueAccessor, OnInit {
 
   }
 
+  // adding save values to from control
+  private _showSavedAnswers() {
+    if ((this.reviewStatus === 'in progress') && (this.doReview)) {
+      if (!this.innerValue) {
+        this.innerValue = {
+          answer: [],
+          comment: ''
+        };
+      }
+      if (this.review.comment) {
+        this.innerValue.comment = this.review.comment;
+        this.comment = this.review.comment;
+      }
+      if (this.review.answer) {
+        this.innerValue.answer = this.review.answer;
```

Comment:
For both `comment` and `answer`, don't need to check if it is empty or not, we can just simply assign `this.review.answer` to `this.innerValue.answer` same for `comment`

Same for other files